README.md

```diff
@@ -451,19 +507,25 @@ The produced binaries still link libc dynamically. If the binary is compiled on
 
 ## Installing Loki from a package
 
-* Docker
+**DISCLAIMER: These packages are not part of this repository or maintained by this project's contributors, and as such, do not go through the same review process to ensure their trustworthiness and security.**
 
-        # Build using all available cores
-        docker build -t loki .
+Packages are available for
 
-        # or build using a specific number of cores (reduce RAM requirement)
-        docker build --build-arg NPROC=1 -t loki .
-
-        # either run in foreground
-        docker run -it -v /loki/chain:/root/.loki -v /loki/wallet:/wallet -p 22022:22022 loki
+* Docker
 
-        # or in background
-        docker run -it -d -v /loki/chain:/root/.loki -v /loki/wallet:/wallet -p 22022:22022 loki
+    ```bash
+    # Build using all available cores
+    docker build -t loki .
+    
+    # or build using a specific number of cores (reduce RAM requirement)
+    docker build --build-arg NPROC=1 -t loki .
+    
+    # either run in foreground
+    docker run -it -v /loki/chain:/root/.bitloki -v /loki/wallet:/wallet -p 18080:18080 loki
```

Comment:
Use port 22022? The same a few lines below.

Reply:
Updated ports